### PR TITLE
Add code of conduct and contribution guideline

### PR DESCRIPTION
**Pull request by skanwal**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Contributing to RNAsum
+
+This outlines how to propose a change to RNAsum.
+For a detailed discussion on contributing to this and other tidyverse packages, please see the [development contributing guide](https://rstd.io/tidy-contrib) and our [code review principles](https://code-review.tidyverse.org/).
```
Comment (pdiakumis) [inline .github/CONTRIBUTING.md:4]:
```suggestion
For a detailed discussion on contributing to this and other packages, please see the [development contributing guide](https://rstd.io/tidy-contrib) and [code review principles](https://code-review.tidyverse.org/) from the tidyverse team.
```

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+You can fix typos, spelling mistakes, or grammatical errors in the documentation directly using the GitHub web interface, as long as the changes are made in the _source_ file. 
+This generally means you'll need to edit [roxygen2 comments](https://roxygen2.r-lib.org/articles/roxygen2.html) in an `.R`, not a `.Rd` file. 
+You can find the `.R` file that generates the `.Rd` by reading the comment in the first line.
+
+## Bigger changes
+
+If you want to make a bigger change, it's a good idea to first file an issue and make sure someone from the team agrees that it’s needed. 
+If you’ve found a bug, please file an issue that illustrates the bug with a minimal 
+[reprex](https://www.tidyverse.org/help/#reprex) (this will also help you write a unit test, if needed).
+See our guide on [how to create a great issue](https://code-review.tidyverse.org/issues/) for more advice.
```
Comment (pdiakumis) [inline .github/CONTRIBUTING.md:17]:
```suggestion
See the tidyverse guide on [how to create a great issue](https://code-review.tidyverse.org/issues/) for more advice.
```

File: README.Rmd [hunk 3]
```diff
@@ -383,3 +383,7 @@ and visualisation is available [here](report_structure.md).
 
 The `results` folder contains intermediate files, including plots and tables
 that are presented in the HTML report.
+
+#### Code of Conduct
+
+The code of conduct can be can be accessed [here](./CODE_OF_CONDUCT.md)
```
Comment (pdiakumis) [inline README.Rmd:389]:
```suggestion
The code of conduct can be accessed [here](./CODE_OF_CONDUCT.md).
```

**Review by pdiakumis — APPROVED** [review]
Thanks @skanwal, LGTM, just a couple changes since we're not really part of the tidyverse :-)

**Comment by skanwal** [comment]
Right! :) Thanks for updates.

**Pull request merged**